app/views/fruits/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render 'boot'%>
+<div class="text-center py-3 bg-info text-white">
```

Comment:
"bg-info" and white have a contrast ratio of 1.96:1, rather than the required 3:1.